packages/warriorjs-core/src/Unit.js

```diff
@@ -106,12 +106,31 @@ class Unit {
   }
 
   /**
+   * Checks if the unit is hostile.
+   *
+   * @returns {boolean} Whether the unit is hostile or not.
+   */
+  isHostile() {
+    return this.hostile;
+  }
+
+  /**
+   * Checks if the unit is friendly.
+   *
+   * @returns {boolean} Whether the unit is friendly or not.
+   */
+  isFriendly() {
+    return !this.hostile;
+  }
+
+  /**
+   * WARNING: Deprecared method
    * Checks if the unit is a captive.
    *
    * @returns {boolean} Whether the unit is a captive or not.
    */
   isCaptive() {
```

Comment:
Let's remove `.isCaptive()` instead of deprecating it as we are still in [initial development phase](https://semver.org/#spec-item-4).